tls/s2n_x509_validator.c

```diff
@@ -405,10 +374,9 @@ s2n_cert_validation_code s2n_x509_validator_validate_cert_chain(struct s2n_x509_
         X509_STORE_CTX_set_time(ctx, 0, current_time);
 
         op_code = X509_verify_cert(ctx);
+        printf("\n Error code: %s\n", X509_verify_cert_error_string(X509_STORE_CTX_get_error(ctx)));
```

Comment:
I'm guessing you didn't mean to leave this in :) Although it would be nice if we could surface it somehow.

Reply:
yes, i did not mean to leave that in. will delete